File: python/tests/test_response_submit_unauthorized.py
#!/usr/bin/env python

import sys
sys.path += ['../']

from epidb.client import EpiDBClient
import config

key = None
data = {
    'user_id': config.user_id,
    'date': '2009-09-09 09:09:09'
}

client = EpiDBClient(key)
client.server = config.server
res = client.response_submit(data)

print res

# vim: ts=4 sts=4 expandtab

File: php/tests/test_response_submit_unauthorized.php
<?

require_once('../EpiDBClient.php');
require_once('config.php');

$key = null;
$data = Array(
    'user_id' => $_user_id,
    'date' => '2009-09-09 09:09:09'
);

$client = new EpiDBClient($key);
$client->server = $_server;
$res = $client->response_submit($data);

var_dump($res);

// vim: ts=4 sts=4 expandtab

